### ISSUE TITLE
Create AdminSet use cases/requirements for PCDM

### ISSUE THREAD

**Issue by azaroth42** [hybox/models#8]
Responsible: @azaroth42 @no-reply 
Accountable: @azaroth42 
Consulted: @mjgiarlo @hannahfrost @anarchivist @jpstroop @escowles @awoods 
Informed: https://github.com/duraspace/pcdm/issues/36

**Issue:**
The PCDM model does not have a distinction between an AdminSet (or other institutional control point for asserting rights/permissions, such as Stanford's APO construct).  Currently this would need to be modeled as a special case for Collection.  If Hydra in a Box requires such a construction, then we need to surface this need early and with sufficient motivation and use cases as to be convincing to the wider community.

**Proposal:**
Responsible to document with the assistance of Consulted the various use cases and requirements in a coherent and comprehensive fashion.  This is to take into account local and known practices, with concrete examples if possible.  The document will then propose a solution for both the model and LDP projection for PCDM, and then pass to engineering for a technical implementation proposal to coordinate.

This issue can be closed when that document has been written and reviewed. The preferred outcome is that the proposals in the document be accepted by the community.

**Comment by azaroth42** [comment]
Propose defer until we have the basics done?

**Comment by anarchivist** [comment]
See [Admin Sets and Admin Policy Objects: Q&A](https://docs.google.com/document/d/1D984LQSGOTQe6cDt9io-_nPkZNxdbXQvdy6YuqYfeOU/edit). Modeling discussion seems to be happening (has already happened?) in the Architecture Working Group.

**Comment by mjgiarlo** [comment]
@anarchivist Has already happened, yup.

**Comment by anarchivist** [comment]
@mjgiarlo The linked Google Doc lists three options for the Admin Set modeling. Did the AWG make a decision? Or are these three aspects of a single model?

**Comment by mjgiarlo** [comment]
@anarchivist Yep, we chose `Directly link (belongs_to)`. Impl is here: https://github.com/projecthydra/curation_concerns/blob/master/app/models/concerns/curation_concerns/admin_set_behavior.rb#L25-L27
